vue.config.js

```diff
@@ -37,13 +37,6 @@ module.exports = {
   publicPath: `${process.env.VUE_APP_PATH}`,
   devServer: {
     proxy: {
-      // this is needed to prevent a CORS error when running locally
-      '/local-keycloak-config-url/*': {
-        target: 'https://dev.bcregistry.ca/business/config/kc/',
-        pathRewrite: {
-          '/local-keycloak-config-url': ''
-        }
-      },
       // this is needed to avoid a PAYBC Not Found error when running locally
       '/status/PAYBC': {
```

Comment:
I think you deleted all the proxy stuff in another project... why not here?

Reply:
Is the '/status/PAYBC' still using in this project?

Reply:
Maybe the sbc-common-components use this.

Reply:
Removed.